src/cmake/wxWidgets/FindwxWidgets.cmake

```diff
@@ -169,8 +169,8 @@ macro(DBG_MSG _MSG)
     "${CMAKE_CURRENT_LIST_FILE}(${CMAKE_CURRENT_LIST_LINE}): ${_MSG}")
```

Comment:
**Question** : this file is processed by cmake only in case of vcpkg use. Why ? (I don't understand the comment in src/CMakeLists.txt : 227)

Reply:
This file is use as a patch for vcpkg because there is currently an issue with `find_package(wxWidgets)` with vcpkg.
See https://github.com/microsoft/vcpkg/issues/4756
There is a PR waiting for review that will maybe remove the need of this specific .cmake.